libsawtooth/src/client/rest.rs

```diff
@@ -269,6 +282,13 @@ struct BlockHeader {
     state_root_hash: String,
 }
 
+/// A struct that represents a state, used for deserializing JSON objects.
+#[derive(Debug, Deserialize)]
+pub struct State {
+    pub address: String,
+    pub data: String,
```

Comment:
These bytes come in as base64 encoded bytes, they should be deserialized from that format.

Reply:
Fixed